### PR TITLE
Updated keyboard shortcut for comments

### PR DESCRIPTION
**Pull request by dhruvipatel14**
Updated wagtail keyboard shortcuts for comments with reference to this discussion: https://github.com/wagtail/wagtail/discussions/12050/#discussioncomment-11489664

Here I have resolved the comments for the **Old PR** : [#12679](https://github.com/wagtail/wagtail/pull/12679)

**Shortcuts before changes** 

This is also applicable if comments are not enabled

comment_not_enabled

**Shortcuts after changes**

If comments are enabled then actions will be updated.

comments_enabled

### CODE REVIEW

**Comment by dhruvipatel14** [comment]
@lb-  Thanks for your valuable feedback on the test cases.

I have updated the test cases and also added the MOD in utils along with the related changes in templatetags.

**Comment by lb-** [comment]
Awesome. I'll try to do a full review locally. 

But pinging @thibaudcolas @laymonage for any other input or reviews also.

File: wagtail/admin/templatetags/wagtailadmin_tags.py [hunk 1]
```diff
@@ -1382,41 +1383,51 @@ def keyboard_shortcuts_dialog(context):
...
                 ),
             ],
             ("actions-model", _("Actions")): [
-                (_("Save changes"), f"{modifier} + s"),
-                (_("Preview"), f"{modifier} + p"),
+                (_("Save changes"), f"{KEYS.MOD} + s"),
+                (_("Preview"), f"{KEYS.MOD} + p"),
+                (
+                    _("Comments"),
+                    f"{KEYS.MOD} + {KEYS.ALT} + m",
```
Comment (lb-) [inline wagtail/admin/templatetags/wagtailadmin_tags.py:1413]:
```suggestion
                    f"{KEYS.CTRL} + {KEYS.ALT} + m",
```

Sorry, this probably got lost in the refactor, it should be ctrl + alt (or ctrl + opt on mac).

https://github.com/wagtail/wagtail/blob/5c30b4211a36cab32727e3a155f70464295e446a/client/src/entrypoints/admin/comments.js#L16-L18

**Review by lb- — APPROVED** [review]
Thank you @dhruvipatel14  I have tested this in a few browsers and emulating different user agents. This is working great.

I have made a few minor refinements to the testing, mainly adding more tests for the 'label' of the comments action, not just the shortcut.

Plus it appears that the mac shortcut was not 100% aligned with the one used.

I will rebase, force push, check the CI & merge.

Note that https://github.com/wagtail/wagtail/issues/11021 is still an issue but it's not related to this PR.

Thank you again for working through the feedback and congrats on your first contribution to Wagtail. I have added you to the contributors list as `Dhruvi Patel`, let me know if you want a different name listed though.


### Screenshots
Screenshot 2025-04-06 at 3 00 23 PM

Screenshot 2025-04-06 at 3 01 00 PM

File: wagtail/admin/templatetags/wagtailadmin_tags.py [hunk 2]
```diff
@@ -1382,41 +1383,51 @@ def keyboard_shortcuts_dialog(context):
...
+                    f"{KEYS.MOD} + {KEYS.SHIFT} + z" if is_mac else f"{KEYS.MOD} + y",
                 ),
             ],
             ("actions-model", _("Actions")): [
-                (_("Save changes"), f"{modifier} + s"),
-                (_("Preview"), f"{modifier} + p"),
+                (_("Save changes"), f"{KEYS.MOD} + s"),
+                (_("Preview"), f"{KEYS.MOD} + p"),
+                (
+                    _("Comments"),
```
[the remark this answers is not in the document]
Comment (dhruvipatel14) [reply wagtail/admin/templatetags/wagtailadmin_tags.py]: Thanks for pointing out that wording, I'll keep it in mind next time. 

I'll look into the older PR and bug.

**Comment by dhruvipatel14** [comment]
Thanks for fixing the CI issue. I wasn't sure about fixing that PostgreSQL and Django version-related check. Are there any specific commands to do that locally?

**Comment by lb-** [comment]
@dhruvipatel14 have a read of this module. 
https://github.com/wagtail/wagtail/blob/8a9d05eef5cf716b028bf47ee5cdfa62abf4facd/wagtail/test/utils/wagtail_tests.py#L10

The WagtailTestUtils was the key, it handles a few edge cases for different databases. 

For you to text locally against different databases, it's a bit tricky but instructions are here. 
https://docs.wagtail.org/en/stable/contributing/developing.html#testing-against-a-different-database
